framework/components/ecl-lang-select-pages/ecl-lang-select-pages.config.js

```diff
@@ -1,40 +1,3 @@
-const data = {
```

Comment:
why did you remove `data`?

Reply:
well, the links were using different modifiers, so I had to split them, but now that we moved the modifiers to twig file, indeed it could be factorized again.